src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/LegacyRazorCompletionEndpointTest.cs

```diff
@@ -405,12 +406,13 @@ public async Task Handle_ProvidesDirectiveCompletionItems()
             // Assert
 
             // These are the default directives that don't need to be separately registered, they should always be part of the completion list.
-            Assert.Contains(completionList.Items, item => item.InsertText == "addTagHelper");
-            Assert.Contains(completionList.Items, item => item.InsertText == "removeTagHelper");
-            Assert.Contains(completionList.Items, item => item.InsertText == "tagHelperPrefix");
+            Assert.Contains(completionList.Items, item => item.InsertText == DirectiveCompletionItemProvider.s_singleLineDirectiveSnippets["addTagHelper"]);
```

Comment:
Up to you, but personally I'd be totally fine not having the snippets be internal, and just switching the check to `item.InsertText.StartsWith("addTagHelper")`. Feels like a more honest check, for example, what if someone accidentally breaks the addTagHelper dictionary entry? No its probably not going to happen, but I like that a test would fail.

Reply:
> not having the snippets be internal

I have a separate test that ensures, that known directives are comming as snippets, so this is not going to happen anyway, so I will add `StartsWith` check as an additional one